src/internal/m365/collection/drive/collections_test.go

```diff
@@ -1719,19 +1412,10 @@ func (suite *CollectionsUnitSuite) TestGet_treeCannotBeUsedWhileIncomplete() {
 	opts.ToggleFeatures.UseDeltaTree = true
 
 	mbh.DrivePagerV = pagerForDrives(drv)
-	mbh.DriveItemEnumeration = mock.EnumerateItemsDeltaByDrive{
-		DrivePagers: map[string]*mock.DriveItemsDeltaPager{
-			id(drive): {
-				Pages: []mock.NextPage{{
-					Items: []models.DriveItemable{
-						driveRootItem(), // will be present, not needed
-						delItem(id(file), parentDir(), rootID, isFile),
-					},
-				}},
-				DeltaUpdate: pagers.DeltaUpdate{URL: id(delta)},
-			},
-		},
-	}
+	mbh.DriveItemEnumeration = mock.DriveEnumerator(
+		mock.Drive(id(drive)).With(
+			mock.Delta(id(delta), nil).With(
+				aPage())))
```

Comment:
is this missing the `delItem` that's in the old version of the code?

Reply:
good catch!